lib/definitions/preview-app-livesync.d.ts

```diff
@@ -34,4 +33,11 @@ declare global {
 		 */
 		link: boolean;
 	}
+
+	interface IPreviewDevicesService extends EventEmitter {
+		connectedDevices: Device[];
```

Comment:
Do we need to expose a public setter? We could expose just a getConnectedDevices method.